src/main/java/persistence/entity/loader/DefaultEntityLoader.java

```diff
@@ -21,7 +26,7 @@ public DefaultEntityLoader(JdbcTemplate jdbcTemplate) {
     public <T> T load(Class<T> clazz, Object id) {
         SchemaMeta schemaMeta = new SchemaMeta(clazz);
         String query = SelectQueryBuilder.builder()
-                .select()
+                .select(schemaMeta.columnNamesWithoutRelation(), schemaMeta.tableName())
```

Comment:
현재는 load action 이 일어날때마다 SchemaMeta 를 생성해 주어야 하는 구조로 보여요.
EntityLoader 생성 시점에 모든 `@Entity` 들의 SchemaMeta 가 의존성으로 주입되어 있었다면
조금 더 유연한 구조가 될 것이라 생각되네요
```suggestion
        SchemaMeta schemaMeta = schemaMetaMap.get(clazz);
        String query = SelectQueryBuilder.builder()
                .select(schemaMeta.columnNamesWithoutRelation(), schemaMeta.tableName())
```
혹은 아래와 같이 쿼리 빌더에 해당 의존성을 주입해줘도 좋을것 같아요
그럼 기존처럼 select 에 컬럼과 테이블 인자를 넘기지 않아도 될 것 같습니다.

```suggestion
        String query = SelectQueryBuilder.builder(schemaMetaMap.get(clazz))
                .select()
```